server/routes/reservationsRouter.js

```diff
@@ -21,5 +30,15 @@ reservationRouter.route('/test')
     }
   })
 
+reservationRouter.route('/:garage_id')
+  .get(async (req, res) => {
+    const list = await getAllReservations({ garage_id: req.params.garage_id, date: req.query.date });
+    if (list) {
+      res.status(200).json(list);
+    } else {
+      res.sendStatus(500);
+    }
+  })
+
```

Comment:
count all reservations for a given garage id, returns the garage id and the number of reservations for the date that was passed in.